### PR TITLE
Proper ordering of the handlers chapters and addition of some examples in the Logging guide

### PR DESCRIPTION
**Pull request by MichalMaler**
This is a follow-up on my Logging guide review (https://github.com/quarkusio/quarkus/pull/34128).
I am fixing some leftover issues  I found and also the issues that was created by my bigger reorg. Also, I am adding some examples for better understatement.

SME (Subject Matter Expert) review need and appreciated.

https://issues.redhat.com/browse/QDOCS-386

### CODE REVIEW

File: docs/src/main/asciidoc/logging.adoc [hunk 1]
```diff
@@ -474,11 +473,10 @@ The Syslog server address and port are specified in the host and port properties
...
 [source, properties]
 ----
-# Per-category Syslog handler
 quarkus.log.syslog.handlers = com.example.MyCategorySyslogHandler
-quarkus.log.syslog.com.example.MyCategorySyslogHandler.level = WARN
-quarkus.log.syslog.com.example.MyCategorySyslogHandler.host = syslog.example.com
-quarkus.log.syslog.com.example.MyCategorySyslogHandler.port = 514
+quarkus.log.syslog.com.example."MyCategorySyslogHandler".level = WARN
+quarkus.log.syslog.com.example."MyCategorySyslogHandler".host = syslog.example.com
+quarkus.log.syslog.com.example."MyCategorySyslogHandler".port = 514
```
Comment (gsmet) [inline docs/src/main/asciidoc/logging.adoc:479]: I don't think this is correct.

**Review by gsmet — CHANGES_REQUESTED** [review]
I added a small comment. This will need a squash before merging.

File: docs/src/main/asciidoc/logging.adoc (hunk 1 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/logging.adoc:479]: @dmlloyd Hello David. Could I ask you for a review and maybe better Syslog examples?

File: docs/src/main/asciidoc/logging.adoc [hunk 2]
```diff
@@ -428,43 +514,30 @@ public final class TestFilter implements Filter {
...
-When enabled, it sends all log events to a syslog server, typically the local syslog server for the application.
-
-For details about its configuration, see the xref:#quarkus-log-logging-log-config_quarkus.log.syslog-syslog-logging[Syslog logging configuration] reference.
 
 
 == Logging configurations examples
 
 This chapter provides examples of frequently used logging configurations.
 
+* An example of a global console handler with additional per-category log level:
```
Comment (MichalMaler) [inline docs/src/main/asciidoc/logging.adoc:539]: @dmlloyd David, here I want to ask you about these leading bulleted sentences. I kinda like the idea to introduce the example a bit, however, I am sure I did not use the proper wording. Are you for tweaking those sentences with your suggestions of for their removal?
Thank you

File: docs/src/main/asciidoc/logging.adoc [hunk 3]
```diff
@@ -385,28 +385,114 @@ WARNING: Enabling pretty printing might cause certain processors and JSON parser
 NOTE: Printing the details can be expensive as the values are retrieved from the caller.
 The details include the source class name, source file name, source method name, and source line number.
 
+
 == Log handlers
 
 A log handler is a logging component responsible for the emission of log events to a recipient.
 Quarkus includes several different log handlers: **console**, **file**, and **syslog**.
 
+The featured examples uses `com.example` as the name of a logging category.
```
Comment (Ladicek) [inline docs/src/main/asciidoc/logging.adoc:394]:
```suggestion
The featured examples use `com.example` as a logging category.
```

File: docs/src/main/asciidoc/logging.adoc [hunk 4]
```diff
@@ -385,28 +385,114 @@ WARNING: Enabling pretty printing might cause certain processors and JSON parser
...
+When enabled, it sends all log events to a syslog server, typically the local syslog server for the application.
+
+* A global configuration example:
++
+[source, properties]
+----
+quarkus.log.syslog.handlers = com.example.GlobalSyslogHandler
+quarkus.log.syslog."com.example.GlobalSyslogHandler".level = INFO
+quarkus.log.syslog."com.example.GlobalSyslogHandler".host = syslog.example.com
+quarkus.log.syslog."com.example.GlobalSyslogHandler".port = 514
```
Comment (Ladicek) [inline docs/src/main/asciidoc/logging.adoc:465]:
```suggestion
quarkus.log.syslog.enable=true
quarkus.log.syslog.app-name=my-application
quarkus.log.syslog.format=%d{yyyy-MM-dd HH:mm:ss} %-5p [%c] (%t) %s%e%n
```

File: docs/src/main/asciidoc/logging.adoc [hunk 5]
```diff
@@ -385,28 +385,114 @@ WARNING: Enabling pretty printing might cause certain processors and JSON parser
...
+----
++
+The INFO level log records for all other categories will also be sent to the same Syslog server.
++
+The Syslog server address and port are specified in the host and port properties.
```
Comment (Ladicek) [inline docs/src/main/asciidoc/logging.adoc:470]:
```suggestion
```

File: docs/src/main/asciidoc/logging.adoc [hunk 6]
```diff
@@ -385,28 +385,114 @@ WARNING: Enabling pretty printing might cause certain processors and JSON parser
...
++
+[source, properties]
+----
+quarkus.log.syslog.handlers = com.example.MyCategorySyslogHandler
+quarkus.log.syslog.com.example."MyCategorySyslogHandler".level = WARN
+quarkus.log.syslog.com.example."MyCategorySyslogHandler".host = syslog.example.com
+quarkus.log.syslog.com.example."MyCategorySyslogHandler".port = 514
+----
++
+The WARN level log records for the `com.example.MyCategory` category will be sent to the specified Syslog server.
```
Comment (Ladicek) [inline docs/src/main/asciidoc/logging.adoc:482]:
```suggestion
```

File: docs/src/main/asciidoc/logging.adoc (hunk 2 above)
Comment (Ladicek) [reply docs/src/main/asciidoc/logging.adoc:539]: The examples are introduced by the "titles" IMHO. In this case, there's

> .Console DEBUG logging except for Quarkus logs (INFO), no color, shortened time, shortened category prefixes

which I think is perfectly enough.

File: docs/src/main/asciidoc/logging.adoc [hunk 7]
```diff
@@ -512,6 +589,8 @@ quarkus.log.category."io.quarkus.category".level=INFO
 quarkus.log.category."io.quarkus.category".handlers=STRUCTURED_LOGGING,STRUCTURED_LOGGING_FILE
 ----
 
+* The use of named handlers, as shown in this example, is typically per-category; however, in this case, it is demonstrated with the `root` category, acting as a global handler:
```
Comment (Ladicek) [inline docs/src/main/asciidoc/logging.adoc:592]:
```suggestion
* The use of named handlers is typically per-category; however, in this case, it is demonstrated with the root category, acting as a global handler:
```

File: docs/src/main/asciidoc/logging.adoc (hunk 1 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/logging.adoc]: @Ladicek OK, thank you somuch, Laďa, for the examples! Patched the examples and pushed the commit.
Cheers

File: docs/src/main/asciidoc/logging.adoc (hunk 2 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/logging.adoc]: OK, I removed that.

File: docs/src/main/asciidoc/logging.adoc [hunk 8]
```diff
@@ -385,28 +385,110 @@ WARNING: Enabling pretty printing might cause certain processors and JSON parser
...
+
+* A global configuration example:
++
+[source, properties]
+----
+quarkus.log.syslog.enable=true
+quarkus.log.syslog.app-name=my-application
+quarkus.log.syslog.format=%d{yyyy-MM-dd HH:mm:ss} %-5p [%c] (%t) %s%e%n
+----
++
```
Comment (Ladicek) [inline docs/src/main/asciidoc/logging.adoc:466]:
```suggestion
```

Missed this, sorry :-)

**Review by Ladicek — APPROVED** [review]

File: docs/src/main/asciidoc/logging.adoc (hunk 1 above)
Comment (Ladicek) [reply docs/src/main/asciidoc/logging.adoc]: Should be fixed now. I actually verified all the examples here using https://github.com/Ladicek/quarkus-tinies/tree/main/logging

**Comment by MichalMaler** [comment]
@gsmet This PR is ready for your check and merge.
I asked @dmlloyd  for a review too; he might have a look at it on Friday.

**Review by gsmet — APPROVED** [review]

**Review by dmlloyd — APPROVED** [review]

**Review by dmlloyd — APPROVED** [review]

**Review by Ladicek — APPROVED** [review]

**Pull request merged**